manifests/a/AndreasWäscher/RepoZ/5.5/AndreasWäscher.RepoZ.locale.en-US.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: AndreasWäscher.RepoZ

```

```suggestion
PackageIdentifier: AndreasWascher.RepoZ
```
Git & the Windows OS works a bit weirdly with special characters in their folder names.